### DISCUSSION TITLE
ScriptReference/ContextMenu

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#564] [Notes]
# ScriptReference/ContextMenu

https://docs.unity3d.com/ScriptReference/ContextMenu.html

**Comment by vertxxyz** [comment]
If you are looking for the context menu that is created when you right-click a serialized property, you want [`EditorApplication.contextualPropertyMenu`](https://docs.unity3d.com/ScriptReference/EditorApplication-contextualPropertyMenu.html).

If you're looking for a static version of this method use [`MenuItem`](https://docs.unity3d.com/ScriptReference/MenuItem.html) with the path as `CONTEXT/ContextTesting/DoSomething` where `ContextTesting` is the name of the class you want to target.
You can add a parameter of type [`MenuCommand`](https://docs.unity3d.com/ScriptReference/MenuCommand.html) to extract the context.
